libsplinter/src/service/scabbard/state.rs

```diff
@@ -358,7 +358,7 @@ impl From<InvalidTransactionResult> for InvalidTransaction {
 pub struct BatchInfo {
     pub id: String,
     pub status: BatchStatus,
-    #[serde(skip_serializing)]
+    #[serde(skip, default = "SystemTime::now")]
```

Comment:
reasoning for this change should be added to the commit

Reply:
addressed